pages/index.js

```diff
@@ -261,6 +266,17 @@ Index.propTypes = {
 const FlexWrap = styled.div`
   display: flex;
   flex-flow: row wrap;
+
+  .mission-statement {
+    color: ${props => props.theme.colors.primaryRed};
+    font-style: italic;
+    font-size: 3rem;
+    letter-spacing: 2px;
+    line-height: 3rem;
```

Comment:
What do you think about increasing the `line-height` a bit? Playing around in the browser, around `4.5rem` felt a little better.